src/coponents/ContactUsModal.tsx

```diff
@@ -0,0 +1,228 @@
+import React, { useState } from 'react';
+import {
+  Text,
+  View,
+  StyleSheet,
+  SafeAreaView,
+  TouchableOpacity,
+  TextInput,
+  Modal,
+} from 'react-native';
+import { Colors } from '../utils/colors';
+import { Dropdown } from 'react-native-element-dropdown';
+import { useMutation } from '@tanstack/react-query';
+import { submitContactUsForm } from '../utils/userService';
+import Toast from 'react-native-toast-message';
+import ModalHeader from './ModalHeader';
+
+type ContactUsModalProps = {
+  isVisible: boolean;
+  onClose: () => void;
+  userToken: string;
+  userEmail: string;
+};
+
+const ContactUsModal: React.FC<ContactUsModalProps> = ({
+  isVisible,
+  onClose,
+  userToken,
+  userEmail,
+}) => {
+  type FormData = {
+    email: string | undefined;
+    support_type: string;
+    message: string;
+  };
+
+  const items = [
+    {
+      label: 'Question',
+      value: 'Question',
+    },
+    {
+      label: 'Bug Report',
+      value: 'Bug Report',
+    },
+    {
+      label: 'Suggestion',
+      value: 'Suggestion',
+    },
+    {
+      label: 'Compliment',
+      value: 'Compliment',
+    },
+  ];
```

Comment:
is there any reason there are two different keys? `label` and `value` seem the same to me.